src/SceneGraphDebugCommandController.spec.ts

```diff
@@ -0,0 +1,210 @@
+import * as sinon from 'sinon';
+import { expect } from 'chai';
+import { SceneGraphDebugCommandController } from './SceneGraphDebugCommandController';
+
+describe('SceneGraphDebugCommandController ', () => {
+    let commandController: SceneGraphDebugCommandController;
+    let execStub: sinon.SinonStub;
+
+    beforeEach(() => {
+        commandController = new SceneGraphDebugCommandController('192.168.1.1');
+        (commandController as any).connectionSuccessful = true;
```

Comment:
you can use array notation to get access to private variables so we don't lose type checking.
```typescript
commandController['connectionSuccessful']= true;
```

Reply:
I was thinking there was a way to do that. I just couldn't remember.